operations/login_info_operation_test.go

```diff
@@ -0,0 +1,24 @@
+package operations
```

Comment:
My understanding of Golang is pretty basic, but there doesn't to be any assertions in this test file. Is this more of a test helper file or are we planning to add assertions later?

Reply:
Ah. This is a Go Example. Its really cool. The comments are the assertion @thoiberg

Reply:
https://blog.golang.org/examples

Reply:
oh cool 😄 . I've approved the PR but apparently I don't have write access to the repo, so you'll have to hit the merge button yourself